os/dbuild.sh

```diff
@@ -140,6 +143,15 @@ function SELECT_OPTION()
 		6|smartfs)
 			BUILD smartfs
 			;;
+		7|trap)
```

Comment:
`trap` is for debugging, not related to build.
So how about changing the menu number `7` to `t`?